java/src/org/openqa/selenium/grid/config/TomlConfig.java

```diff
@@ -19,30 +19,27 @@
 
 import com.google.common.collect.ImmutableList;
 import com.google.common.collect.ImmutableSortedSet;
-import io.ous.jtoml.JToml;
-import io.ous.jtoml.ParseException;
-import io.ous.jtoml.Toml;
-import io.ous.jtoml.TomlTable;
 import java.io.IOException;
 import java.io.Reader;
 import java.nio.file.Files;
 import java.nio.file.Path;
-import java.util.Collection;
-import java.util.List;
-import java.util.Optional;
-import java.util.Set;
+import java.util.*;
+import java.util.stream.Collectors;
+
 import org.openqa.selenium.internal.Require;
+import org.tomlj.*;
 
 public class TomlConfig implements Config {
 
-  private final Toml toml;
+  private final TomlParseResult toml;
 
   public TomlConfig(Reader reader) {
     try {
-      toml = JToml.parse(reader);
+      toml = Toml.parse(reader);
+      System.out.println(toml.toToml());
```

Comment:
Please remove sys out statements

Reply:
I will remove when I complete the functionality of needed and pass all the tests.

Reply:
My bad. I didnt realise that this was a draft PR :)